strapi/strapi
# [DEITS] restore mechanism

### What does it do?

- performs a restore before starting an import
- deletes all data except for some default exceptions (admin related stuff and api tokens)
- moved strategy handling from engine to destination provider

### How to test it?

- run a strapi app and add some data
- run `yarn strapi export` to get and export file
- run `yarn strapi import strapi-backup`

After that run the strapi app and data should be gone
